spec/features/brexit_checker/question_results_spec.rb

```diff
@@ -58,6 +61,16 @@ def then_i_see_citizens_results_only
     and_i_should_not_see_a_tourism_action
   end
 
+  def and_i_should_see_share_links
+    current_url = CGI.escape(page.current_url)
+    expect(page).to have_css(".gem-c-share-links")
```

Comment:
This `gem-c-share-links` class is generated by the `govuk_publishing_components`, which means these tests will fail if this class name ever changes there, even though nothing is broken. Maybe we could make these tests more flexible, and only check for the presence on the page of the URLs?

Reply:
This is fairly standard across GOV.UK - we don't expect classes in our shared components  to change regularly. But fair point, we don't really need to use them in this case. Will amend 👍